### ISSUE TITLE
Error when running on supervisor

### ISSUE THREAD

**Issue by hallowcard13** [soketi/soketi#410]
Manually starting works fine, but using Supervisor, I'm getting the error below

```
Error: This version of µWS is not compatible with your Node.js build:

Error: Cannot find module './uws_linux_x64_64.node'
    at module.exports (/usr/local/lib/node_modules/@soketi/soketi/node_modules/uWebSockets.js/uws.js:22:9)
    at Object.<anonymous> (/usr/local/lib/node_modules/@soketi/soketi/node_modules/uWebSockets.js/uws.js:24:3)
    at Module._compile (internal/modules/cjs/loader.js:778:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:789:10)
    at Module.load (internal/modules/cjs/loader.js:653:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
    at Function.Module._load (internal/modules/cjs/loader.js:585:3)
    at Module.require (internal/modules/cjs/loader.js:692:17)
    at require (internal/modules/cjs/helpers.js:25:18)
    at Object.<anonymous> (/usr/local/lib/node_modules/@soketi/soketi/dist/server.js:17:13)
[PM2][Initialization] Environment variable HOME (Linux) or HOMEPATH (Windows) are not set!
[PM2][Initialization] Defaulting to /etc/.pm2
/usr/local/lib/node_modules/@soketi/soketi/node_modules/uWebSockets.js/uws.js:22
                throw new Error('This version of µWS is not compatible with your Node.js build:\n\n' + e.toString());
                ^

```

```
Node = 16.14.0
NPM = 8.5.3
OS = Ubuntu 20.04
```

**Comment by hallowcard13** [comment]
Update:
I'm using now an NVM to install node and npm, still, manual  starting on terminal or CLI is working fine, but on supervisor, i'm getting an error now that 'Cant find command soketi

**Comment by hallowcard13** [comment]
Update:
fixed the issue, when deploying on ubuntu 20.04  use this link to install node js https://github.com/nodesource/distributions
Also on supervisor, I have used "root" as the user.
